src/components/Modals/Send/views/Address.tsx

```diff
@@ -29,6 +30,8 @@ import { SendRoutes } from '../Send'
 
 export const Address = () => {
   const [isValidatingEnsName, setisValidatingEnsName] = useState(false)
+  const [isValidatingCosmosAddress, setIsValidatingCosmosAddress] = useState(false)
+  const isValidating = isValidatingEnsName || isValidatingCosmosAddress
```

Comment:
If anyone has idea of a better naming for this, I'll be happy to change. Technically, `WAValidating` is also validation but is synchronous and very fast, effectively instant, so there's no need to have it as part of the `isValidating` state.
From a semantic perspective, this is a white lie as it really means "I'm currently validating in an async way (ENS name/Cosmos address validity from unchained)"